OracleTuxedo/core/dockerfiles/12.1.3/install.sh

```diff
@@ -8,11 +8,8 @@
 #
 #   and that these files can be found in the in /home/oracle/Downloads
 #	tuxedo121300_64_Linux_01_x86.zip	- Tuxedo installation kit from Oracle
-#	p19927652_121300_Linux-x86-64.zip	- Tuxedo lastest patch kit from Oracle - RP011
-#						  actual file name may be different
 #
-# Usage:  sh tuxedo12.1.3_silent_install.sh [patch-filename [tuxedo-installer-filename]]
-#	patch-filename defaults to the latest patch file in the current directory
+# Usage:  sh tuxedo12.1.3_silent_install.sh [tuxedo-installer-filename]
```

Comment:
Putting this comment here, but it's slightly more generic: what's the point of `install.sh`? Why not just run these commands in the `Dockerfile` itself? That would make the build less of a black box and potentially easier to debug in the case of potential failures.